docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1749 @@
+| **Possible values** | TCP Stream socket (`"tcp://127.0.0.1:26658"`)      |
+|                     | Unix domain socket (`"unix:///var/run/abci.sock"`) |
+|                     | `"kvstore"`                                        |
+|                     | `"persistent_kvstore"`                             |
+|                     | `"noop"`                                           |
+
+When the ABCI application is written in a different language than Golang, (for example the
+[Nomic binary](https://github.com/nomic-io/nomic) is written in Rust) the application can open a TCP port or create a
+UNIX domain socket to communicate with CometBFT, while CometBFT runs as a separate process.
+
+The [abci](#abci) parameter is used in conjunction with this parameter to define the protocol used for communication.
+
+In other cases, (for example in the [Gaia binary](https://github.com/cosmos/gaia)) CometBFT is imported as a library
+and the configuration entry is unused.
+
+For development and testing, the built-in ABCI applications can be used without additional processes running.
+
+<!--- Todo: describe the built-in applications and their use --->
+
+### moniker
+A custom human-readable name for this node.
+```toml
+moniker = "my.host.name"
+```
+
+| Value type          | string                                                   |
+|:--------------------|:---------------------------------------------------------|
+| **Possible values** | any human-readable string                                |
+
+The main use of this entry is to keep track of the different nodes in a local environment. For example the `/status` RPC
+endpoint will return the node moniker in the `.result.moniker` key.
+
+Monikers do not need to be unique. They are for local administrator use and troubleshooting.
+
+Nodes on the peer-to-peer network are identified by `nodeID@IP:port` as discussed in the
+[node_key.json](node_key.json.md) section.
+
+### db_backend
+The chosen database backend for the node.
+```toml
+db_backend = "goleveldb"
+```
+
+| Value type          | string        | dependencies  | GitHub                                           |
+|:--------------------|:--------------|:--------------|:-------------------------------------------------|
+| **Possible values** | `"goleveldb"` | pure Golang   | [goleveldb](https://github.com/syndtr/goleveldb) |
+|                     | `"cleveldb"`  | requires gcc  | [leveldb](https://github.com/google/leveldb)     |
+|                     | `"boltdb"`    | pure Golang   | [bbolt](https://github.com/etcd-io/bbolt)        |
+|                     | `"rocksdb"`   | requires gcc  | [grocksdb](https://github.com/linxGnu/grocksdb)  |
+|                     | `"badgerdb"`  | pure Golang   | [badger](https://github.com/dgraph-io/badger)    |
+|                     | `"pebbledb"`  | pure Golang   | [pebble](https://github.com/cockroachdb/pebble)  |
+
+During the build process, only the `goleveldb` library is built into the binary.
```

Comment:
Is this also true for the binaries offered on GitHub?

Reply:
Probably adding "by default" somewhere in this sentence will address @cason's concern